### ISSUE TITLE
What is the status of this project?  Is it production ready or pre-Alpha?

### ISSUE THREAD

**Issue by frederickjh** [coldfrontlabs/fserver#46]
Hi!

I am looking to setup a feature server. I see on drupal.org that the D7 is not production ready but the D6 version can server D7 features. I was wondering what the status of your project is as I saw some one else comment that they had a lot of errors after install in the issue que recently.

Thanks for a quick reply!

Frederick Henderson

**Comment by frederickjh** [comment]
Well a fresh install of Drupal 7.36 + enabling the modules and needed dependencies + enabling the UI + dependencies, I find that the views are broken. I am also seeing a database errors. Below are a few screenshots of the errors. I hope this helps to fix these. If you need more info let me know quick as I will most likely be blowing this install away and replacing it with a D6 fserver install tomorrow.

God Bless!
Frederick Henderson

![screenshot from 2015-04-15 22 22 42-d7-fserver_ui_broken_views](https://cloud.githubusercontent.com/assets/864159/7168747/17693ab0-e3c0-11e4-9fdf-3970b67670e7.png)

![d7-fserver_ui_broken_views-projects_site-install](https://cloud.githubusercontent.com/assets/864159/7168716/cf739778-e3bf-11e4-9819-5a83c9cf22a3.png)

![d7-fserver_ui_broken_views-releases_site-install](https://cloud.githubusercontent.com/assets/864159/7168721/d779c71c-e3bf-11e4-8f8b-e369da5ce4a1.png)

![screenshot from 2015-04-15 22 31 21-fserver_ui_project-list-error_message](https://cloud.githubusercontent.com/assets/864159/7168847/c02b0dfe-e3c0-11e4-894b-bf9519dd3fae.png)

**Comment by frederickjh** [comment]
Not sure if it makes any difference but I use the <b>minimal</b> install profile when installing D7.

**Comment by semyondragunov** [comment]
https://github.com/coldfrontlabs/fserver/issues/45

**Comment by minorOffense** [comment]
Yeah the default panels UI stuff needs to be created. The module itself is stable, the XML work, releases work, projects, project pages, release pages, etc... Just need someone to recreate a default UI to view lists of projects, their fields and what not.

Essentially, all modules are complete except for fserver_ui . We're open to anyone who wants to create a default UI for fserver. Feel free to make a pull request.

**Comment by fugazi73** [comment]
problem, unfortunately, not yet been solved
